M2/Macaulay2/packages/PencilsOfQuadrics.m2

```diff
@@ -9,25 +9,26 @@ viewHelp "PencilsOfQuadrics"
 needsPackage"CompleteIntersectionResolutions"
 loadPackage("PencilsOfQuadrics", Reload=>true)
 peek loadedFiles
-
 ///
+
      newPackage(
              "PencilsOfQuadrics",
-             Version => "0.9", 
-             Date => "June 17, 2020",
+    	     Version => "1.0",
+    	     Date => "October 10, 2024",
              Authors => {{Name => "Frank-Olaf Schreyer", 
                        Email => "schreyer@math.uni-sb.de", 
                        HomePage => ""},
              	         {Name => "David Eisenbud", 
                        Email => "de@msri.org", 
-                       HomePage => "https://www.msri.org/~de"},
+                       HomePage => "www.msri.org/~de"},
 		       {Name => "Yeongrak Kim",
-			   Email => "kim@math.uni-sb.de",
-			   HomePage => "https://sites.google.com/view/yeongrak"}
+    	    	    	   Email => "yeongrak.kim@pusan.ac.kr",
+			   HomePage => "sites.google.com/view/yeongrak"}
```

Comment:
Removing the "https://" from these two `HomePage` fields is causing a failing test:

```
cd /home/runner/work/M2/M2/M2/BUILD/build/usr-dist/common/share/doc/Macaulay2 && find . -name \*.html | xargs -r ls -rt | LD_LIBRARY_PATH="/home/runner/work/M2/M2/M2/BUILD/build/usr-host/lib:$LD_LIBRARY_PATH" xargs "/home/runner/work/M2/M2/M2/BUILD/build/Macaulay2/html-check-links"/html-check-links --no-absolute-links
./PencilsOfQuadrics/html/index.html:124:9: error: broken link: ../../../../../../www.msri.org/~de
./PencilsOfQuadrics/html/index.html:126:9: error: broken link: ../../../../../../sites.google.com/view/yeongrak
make[2]: *** [Makefile:62: run-html-check-links] Error 123
```